src/service/layers-impl.js

```diff
@@ -1240,11 +1247,15 @@ export class LayoutElement {
 
     // Now, recursively measure all child nodes, to since they've probably been
     // invalidated by the parent changing.
+    // TODO(jridgewell): When do children need to be remeasured? Could we skip
+    // for only size changes? Or only position changes?
```

Comment:
Now that AMP elements are layers, why do we need to remeasure children when a parent's position changes?

Reply:
On the off chance that whatever caused the parent layer to change position also affects the measurements of children. Imagine an amp-experiment applying to the html elmenet causing a cascade of changes.

We may need to always check.

Reply:
I'm actually going to undo this optimization. I'd rather be safe for now, and we can see what implies a needed remeasure later.